pkg/cmdlets/auth/login.go

```diff
@@ -178,3 +182,27 @@ func promptToken(credentials *mono_models.Credentials, out output.Outputer, prom
 
 	return nil
 }
+
+// AuthenticateWithDevice attempts to authenticate this device with the Platform.
+func AuthenticateWithDevice(out output.Outputer) error {
+	deviceCode, err := model.RequestDeviceAuthorization()
+	if err != nil {
+		return err
+	}
+	out.Notice(locale.Tr("auth_device_verify_security_code", *deviceCode.UserCode))
+	err = OpenURI(*deviceCode.VerificationURIComplete)
+	if err != nil {
+		logging.Error("Could not open browser: %v", err)
+		out.Notice(locale.Tr("err_browser_open", *deviceCode.VerificationURIComplete))
```

Comment:
Is there a use-case for there being an error but us still wanting to continue? ie. why not return this error?

Reply:
This is not an error that would prevent authentication. The user is prompted here to manually copy-paste the URL into their browser and authorize the request. In the meantime, we can still poll for authorization.

Reply:
Ahh I see, makes sense! In that case please change `logging.Error` to `logging.Warning`.

Reply:
Okay.